HW02_A_Graph-Fails.Rmd

```diff
@@ -46,12 +56,29 @@ ggplot(data = mpg) +
 
 ```
 
+#CORRECTED CODE for Graph 2
+```{r}
+ggplot(mpg,aes(displ,hwy))+
+  geom_point(color="blue")
+```
 
 
 ### Graph Fail 3
 Two mistakes in this graph. First, I wanted to make the the points slightly bolder, but changing the alpha to 2 does nothing. What does alpha do and what does setting it to 2 do? What could be done instead if I want the points slightly bigger? 
 
+#ANSWER for Graph 3 - 1st mistake
+Alpha is an argument that controls the opacity of the points so setting it to 2 will make the points appear at 20% transparency. To make the points slightly bigger, I would use size instead of alpha. 

```

Comment:
I thought that I remembered alpha having a range from 0 to 1 (0 being completely transparent and 1 being opaque).  I did a quick google search and found this (https://ggplot2.tidyverse.org/reference/scale_alpha.html) which explains a little about alpha with ggplot2 in R.  

I am not quite sure what alpha = 2 does.  When I change this and generate the figure nothing seems to change on my end, unless you are getting some different results?

Reply:
I agree that alpha should be set between 0 to 1, with 0 referring to completely transparent. I think you may get error message if alpha is set to 2?

Reply:
Woops, thanks for pointing it out guys! I went back to check my notes and yes, you guys are right about alpha being a value from 0 - 1. For some reasons, I saw the number 2 but in my head, I kept thinking of it as 0.2, thus the 20% transparency point. 

I went back to my codes to double check. Putting the alpha = 0.2 makes the points appear at 20% transparency. Putting the alpha = 2 does not give me an error message, instead the codes were able to run but as Ariana mentioned, the figure didn't seem to change anything.